helm/fiftyone-teams-app/values.yaml

```diff
@@ -403,7 +403,7 @@ teamsAppSettings:
     FIFTYONE_APP_ALLOW_MEDIA_EXPORT: true
     # -- The recommended fiftyone SDK version that will be displayed in the
     # install modal (i.e. `pip install ... fiftyone==0.11.0`).
-    FIFTYONE_APP_TEAMS_SDK_RECOMMENDED_VERSION: 0.15.0
+    FIFTYONE_APP_TEAMS_SDK_RECOMMENDED_VERSION: 0.15.1
```

Comment:
I would have expected that this change would have updated the default value in `helm/fiftyone-teams-app/README.md` (handled by the pre-commit check helm-docs)?